low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -139,23 +147,20 @@ class SSPLBundleGenerate(object):
             with tarfile.open(self.localTempPath+file_name, "w:gz") as tar:
                 for key, value in tarfile_data.items():
                     if os.path.exists(key):
-                        tar.add(key, arcname=value)
+                        try:
+                            tar.add(key, arcname=value)
+                        except IOError as err:
+                            print("Unable to include %s logs with an error %s" % (key, err))
```

Comment:
all these prints are not seen when invoked through cortx-cli, in which case, can we explore redirecting all these to cortx-cli log or redirect all these logs to sspl-support-bundle-<bundle-id>. Check with @pawankumarsrivastava if any common folder to retain support-bundle logs.

Reply:
@satish-seagate @cdeshmukh There is no common place for the storage of support bundle logs. Since the command is invoked with cortxcli so all the logs are written in cortxcli logs.
Location - /var/log/seagate/csm/csm_cli.log

Reply:
@pawankumarsrivastava  sspl support bundle error logs didnt show up in csm_cli.log when we last checked during live webex session for EOS-18667(padova tech) issue.

Reply:
We will address this comment through https://jts.seagate.com/browse/EOS-19080